src/components/accordion-item/accordion-item.tsx

```diff
@@ -190,12 +220,15 @@ export class AccordionItem implements ConditionalSlotComponent {
               role="button"
               tabindex="0"
             >
-              {this.icon ? iconEl : null}
-              <div class={CSS.headerText}>
-                <span class={CSS.heading}>{this.heading || this.itemTitle}</span>
-                {this.itemSubtitle || this.description ? (
-                  <span class={CSS.description}>{this.description || this.itemSubtitle}</span>
-                ) : null}
+              <div class={CSS.headerContainer}>
+                {this.iconStart ? iconStartEl : null}
+                <div class={CSS.headerText}>
+                  <span class={CSS.heading}>{this.heading || this.itemTitle}</span>
+                  {this.itemSubtitle || this.description ? (
```

Comment:
Is there a follow up refactor issue to make `description` the source of truth so that these two don't compete?

Reply:
`description` is a string and doesn't require a follow-up refactor IMO  as we don't append both the attribute to DOM if one of them is true. Even when the user parses both `description` & `itemSubtitle`, the newer prop takes precedence.

Reply:
Yeah that makes sense. I was just thinking a const would be nice so that `this.description || this.itemSubtitle` isn't repeated.